shop3/looping-tasks/loop-extensions.lisp

```diff
@@ -34,25 +34,24 @@
 
 (defmethod unfold-loop-task ((domain looping-domain)
                              ess-search-state)
-;  (format t "~%Unfolding now...")
```

Comment:
We should probably either restore the commenting-out, or make this dependent on the value of `*verbose*`.

Similarly for other `format` statements.